📝 docs(release): Update baize release docs (#5644)
DaoCloud/DaoCloud-docs: docs/en/docs/baize/intro/release-notes.md

This page lists the Release Notes for AI Lab,
so that you can learn its evolution path and feature changes.

## 2024-09-30

### v0.9.0

#### Features

- **Added** Added a new data management sub-module “Data Annotation” which can manage data annotation capabilities for
  mainstream data types.
- **Added** Added a new model management sub-module “Model List” which supports quick data import.

> - **Added** When creating a “Data Set”, the PVC storage space size can be specified.

Reviewer: ```suggestion
- **Added** When creating a “Dataset”, the PVC storage space size can be specified.
```

> - **Added** Supports one-click restart of “training tasks”.

Reviewer: ```suggestion
- **Added** Supports one-click restart of “Training jobs”.
```

- **Added** When using vGPU resources, the GPU card type can be specified.
- **Added** The base image of “baize-notebook” has been upgraded to v0.9.0.
- **Improved** Supports global alerts while ensuring data availability when there is a cluster exception.

## 2024-08-31

### v0.8.0

#### Features

> - [Beta] **Added** Support manually saving as an image while a `Notebook` is running (depends on the image repository
>   module).
> - [Beta] **Added** Support automatically saving as an image when a `Notebook` is closed (depends on the image repository

Reviewer 2: ```suggestion
- **Added** [Beta] support manually saving as an image while a `Notebook` is running (depends on the image repository
  module).
- **Added** [Beta] support automatically saving as an image when a `Notebook` is closed (depends on the image repository
```

Author: 因为发现上个月的英文没提交，所以顺便补充了。
